dtfs-central-api/src/v1/controllers/utilisation-report-service/get-selected-fee-records-details.controller/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { CurrencyAndAmount, FeeRecordEntity, ReportPeriod, SelectedFeeRecordDetails, SelectedFeeRecordsDetails } from '@ukef/dtfs2-common';
+import { getBankNameById } from '../../../../repositories/banks-repo';
+import { NotFoundError } from '../../../../errors';
+import { mapFeeRecordEntityToReportedFees, mapFeeRecordEntityToReportedPayments } from '../../../../mapping/fee-record-mapper';
+
+const mapFeeRecordEntityToSelectedFeeRecordDetails = (feeRecordEntity: FeeRecordEntity): SelectedFeeRecordDetails => ({
+  id: feeRecordEntity.id,
+  facilityId: feeRecordEntity.facilityId,
+  exporter: feeRecordEntity.exporter,
+  reportedFee: mapFeeRecordEntityToReportedFees(feeRecordEntity),
+  reportedPayment: mapFeeRecordEntityToReportedPayments(feeRecordEntity),
+});
+
+const getTotalReportedPayments = (feeRecords: SelectedFeeRecordDetails[]): CurrencyAndAmount => ({
+  currency: feeRecords[0]?.reportedPayment.currency,
+  amount: feeRecords.map((feeRecord) => feeRecord.reportedPayment.amount).reduce((total, currentAmount) => total + currentAmount),
+});
+
+export const mapToSelectedFeeRecordDetails = async (
+  bankId: string,
+  reportPeriod: ReportPeriod,
+  selectedFeeRecordEntities: FeeRecordEntity[],
```

Comment:
That's an interesting point, since it was just test readability in this case I have updated the tests to be more readable as I don't generally like packaging up parameters just to immediately unpack them, but that is an interesting case